variables.tf

```
@@ -24,6 +24,12 @@ variable "user_data" {
   default     = "false"
 }
 
+variable "additional_user_data" {
+  description = "Bash code to APPEND to the default EC2 user_data. NOTE: Use this OR `user_data`, not both."
```

Maybe it's helpful to specify which takes precedence in case both are given.
```suggestion
  description = <<-EOT
    Bash code to APPEND to the default EC2 user_data. NOTE: If `user_data` is
    specified, this will be ignored.
  EOT
```